marlowe/src/Language/Marlowe/Client.hs

```diff
@@ -157,24 +157,16 @@ type MarloweContractState = LastResult
 
 marloweFollowContract :: Contract ContractHistory MarloweFollowSchema MarloweError ()
 marloweFollowContract = awaitPromise $ endpoint @"follow" $ \params -> do
-    slot <- currentSlot
-    let client@StateMachineClient{scInstance} = mkMarloweClient params
-    let inst = SM.typedValidator scInstance
-    let address = Scripts.validatorAddress inst
+    let client = mkMarloweClient params
     let go [] = pure InProgress
         go (tx:rest) = do
             res <- updateHistoryFromTx client params tx
             case res of
                 Finished   -> pure Finished
                 InProgress -> go rest
 
-    AddressChangeResponse{acrTxns} <- awaitPromise $ addressChangeRequest
-                AddressChangeRequest
-                { acreqSlotRangeFrom = 0
-                , acreqSlotRangeTo = slot
-                , acreqAddress = address
-                }
-    go acrTxns >>= checkpointLoop (follow client params)
+    go [] >>= checkpointLoop (follow client params)
```

Comment:
I kept the `go` because if I deleted it, I would need to delete other unused functions like `updateHistoryFromTx` and `findInput`. However, I wanted to keep them for now and let Alex decide to remove it if necessary. Or I could comment them out. What do you think?

Reply:
ok, fair enough. @nau  can delete them if he wants.